pom.xml

```diff
@@ -197,41 +244,16 @@
         </dependency>
     </dependencies>
 
-    <dependencyManagement>
-        <dependencies>
-            <dependency>
-                <groupId>org.jdbi</groupId>
-                <artifactId>jdbi3-bom</artifactId>
-                <type>pom</type>
-                <version>3.14.1</version>
-                <scope>import</scope>
-            </dependency>
-        </dependencies>
-    </dependencyManagement>
-
     <build>
         <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.8.1</version>
-                <configuration>
-                    <source>11</source>
-                    <target>11</target>
-                    <encoding>UTF-8</encoding>
-                    <compilerArgs>
-                        <arg>-parameters</arg>
```

Comment:
We need this compiler arg, you can't remove this

Reply:
Re-added it.

Reply:
that flag makes the compiler include method params names in the LVT, allowing jdbi, the DB mapper we use, to automagically map stuff to DB tables by looking at the names of the params

Reply:
Ah for that it is used :D